ethereum/core/src/main/java/org/hyperledger/besu/ethereum/debug/TraceOptions.java

```diff
@@ -14,30 +14,7 @@
  */
 package org.hyperledger.besu.ethereum.debug;
 
-public class TraceOptions {
-
-  private final boolean traceStorage;
-  private final boolean traceMemory;
-  private final boolean traceStack;
-
-  public static final TraceOptions DEFAULT = new TraceOptions(true, false, true);
-
-  public TraceOptions(
-      final boolean traceStorage, final boolean traceMemory, final boolean traceStack) {
-    this.traceStorage = traceStorage;
-    this.traceMemory = traceMemory;
-    this.traceStack = traceStack;
-  }
-
-  public boolean isStorageEnabled() {
-    return traceStorage;
-  }
-
-  public boolean isMemoryEnabled() {
-    return traceMemory;
-  }
-
-  public boolean isStackEnabled() {
-    return traceStack;
-  }
+public record TraceOptions(
```

Comment:
is it possible to have the record modification in the main branch in order to reduce the amount of conflicts in the future ?